### DISCUSSION TITLE
Impact of new routing proposal on mdsvex

### DISCUSSION THREAD

**Discussion by maiertech** [pngwn/MDsveX#465] [General]
In view of Rich Harris's routing proposal (https://github.com/sveltejs/kit/discussions/5748) that might be implemented as early as next week, what will be the impact on mdsvex?

**Comment by ashfidable** [comment]
Looks like it still works. All you have to do is make the folder named as blog slug like hello-world and create +page.md to work. 

~~Is there any way to put all md files into another directory and get compiled data and then render it? The compile part is the issue for me and I don't understand it well. 
I don't really enjoy creating +page.md route files and folders for blogging.~~ got this working 🥳
